src/ralph/assets/utils.py

```diff
@@ -1,7 +1,34 @@
 # -*- coding: utf-8 -*-
+from django.conf import settings
 from django.db import transaction
 
 
+class DNSaaSPublisherMixin:
+    """Generate data formatted for DNSaaS auto txt update"""
+    def get_auto_txt_data(self):
+        data = []
+        for purpose_name, content in  (
+            ('class_name', self.configuration_path.class_name if self.configuration_path else ''),  # noqa
+            ('module_name', self.configuration_path.module.name if self.configuration_path else ''),  # noqa
```

Comment:
please add another entry `configuration_path` pointing to `self.configuration_path.path`